vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt

```diff
@@ -83,6 +83,7 @@ class UiAllScreensSanityTest {
     // 2021-04-08 Testing 429 change
     @Test
     fun allScreensTest() {
+        uiTestBase.enableDeveloperMode()
```

Comment:
I would prefer to not do that at the beginning of the main test, but rather add a dedicated `fun` to test all developer mode features at the end of this test (there are other screens we could navigate to, devtools, account data, key requests (and we have seen recently that this last one can crash, you fixed it 💯 )), and eventually optionally at the end of this `fun` disable the developer mode.

Reply:
a fair point, I definitely took the shortcut here by editing the preferences directly 

will close and reopen with the tests interacting with the real dev mode button :+1:

Reply:
Thanks!